src/java/org/apache/cassandra/tools/nodetool/Status.java

```diff
@@ -98,6 +105,7 @@ public void execute(NodeProbe probe)
         unreachableNodes = probe.getUnreachableNodes(true);
         hostIDMap = probe.getHostIdMap(true);
         epSnitchInfo = probe.getEndpointSnitchInfoProxy();
+        cmsProxy = hasCms ? probe.getCMSOperationsProxy() : null;
```

Comment:
same as above - maybe just grab which nodes are actually in the CMS here and use that set when outputting below instead of querying for each node?